packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: System Log events for rate limits
+category: rate limits
+---
+
+# System Log events for rate limits
+
+The Okta [System Log](/docs/reference/api/system-log/) records system events related to your organization to provide an audit trail that you can use to understand platform activity and to diagnose problems. There are a few standard System Log events that you may encounter when Okta is enforcing [rate limits](/docs/reference/rate-limits/) on specific operations.
+
+* `system.operation.rate_limit.violation`<br>
+This event type is sent once per rate limiting period when a request is rejected for exceeding a rate limit. For example, if you exceed your calls per minute quota (60 seconds), then one event is sent within that 60 seconds.
```

Comment:
"For example, if the rate limit that has been exceeded has a reset period of one minute, then one event of this type will be emitted during that period for the applicable scope."
... or something like that